draft-ajitomi-cose-cose-key-jwk-hpke-kem.md

```diff
@@ -47,12 +52,6 @@ This document defines an additional key parameter and a new key type for CBOR Ob
 Hybrid Public Key Encryption (HPKE) {{RFC9180}}, published by the Internet Research Task Force (IRTF), has already been adopted in several communication protocol specifications such as TLS Encrypted Client Hello (ECH), Oblivious DNS over HTTPS (ODoH) and Oblivious HTTP (OHTTP).
 HPKE itself is communication protocol independent and can be widely used as a standard scheme for public key based end-to-end encryption in various applications, not only in communication protocols.
 
-In HPKE, the sender of a ciphertext needs to know in advance not only the recipient public key, but also the HPKE mode, the KEM associated with the key, and the set of supported KDF and AEAD algorithms.
-The data structure of this information (hereafter referred to as HPKE key configuration information) is defined in each communication protocol specification that uses HPKE.
-For example, the ECH defines it as a structure called HpkeKeyConfig.
-When using HPKE in an application, it is necessary to define the data structure corresponding to the HpkeKeyConfig and how the information is transferred from the recipient to the sender.
-If the data structure and the publication method for the HPKE key configuration information were standardized, it would be easier to use HPKE in applications.
-
```

Comment:
This is a clarifying question, but I'd like to know the difference between "for key use restriction" and "for end-to-end application algorithm negotiation." I believe this removal is related to the concepts you suggested. I can agree with the concepts below, but I don't understand the necessity of removing the entire sentences.
When using HPKE in an application, it is necessary to define the data structure corresponding to the HpkeKeyConfig and how the information is transferred from the recipient to the sender.
If the data structure and the publication method for the HPKE key configuration information were standardized, it would be easier to use HPKE in applications.
```
the alg and other params are not mandatory, the alg parameter and HPKE key config are for key use restriction, end-end application algorithm negotiation is not a primary purpose.
```

For instance, would the following sentences be acceptable to you?

```suggestion
In HPKE, the recipient of a ciphertext needs to restrict the use of its public key by the sender. Specifically, the recipient expects the public key to be used with a specific HPKE mode, a specific KEM, and specific KDF and AEAD algorithms that the recipient supports.
The data structure of this information (hereafter referred to as HPKE key configuration information) is defined in each communication protocol specification that uses HPKE.
For example, the ECH defines it as a structure called HpkeKeyConfig.
When using HPKE in an application, it is desirable to define the data structure corresponding to the HpkeKeyConfig and how the information is transferred from the recipient to the sender.
If the data structure and the publication method for the HPKE key configuration information were standardized, it would be easier to use HPKE in applications.
```